pipelines/Jenkinsfile

```diff
@@ -0,0 +1,240 @@
+        yaml """
+apiVersion: v1
+kind: Pod
+metadata:
+  name: jenkins-node-pg-0
+  namespace: jenkins
+  labels:
+    app.kubernetes.io/name: jenkins-node-pg
+spec:
+  volumes:
+    - name: rsa-keys
+      configMap:
+        name: rsa-keys
+        defaultMode: 384
+    - name: docker-sock
+      hostPath:
+        path: /var/run/docker.sock
+        type: ''
+  containers:
+    - name: compiler
+      image: etendo/compiler_jenkins:1.0.6
+      ports:
+        - name: ssh
+          containerPort: 22
+          protocol: TCP
+        - name: visualvm
+          containerPort: 8000
+          protocol: TCP
+      resources:
+        requests:
+          cpu: 1072m
+          memory: 4000Mi
+        limits:
+          cpu: 2072m
+          memory: 6500Mi
+      volumeMounts:
+        - name: rsa-keys
+          mountPath: /root/.ssh/
+        - name: docker-sock
+          mountPath: /var/run/docker.sock
+      lifecycle:
+        postStart:
+          exec:
+            command:
+              - bash
+              - '-c'
+              - >-
+                chmod a+x /var/run/docker.sock && rm
+                /etc/apt/sources.list.d/pgdg.list || echo 0 && apt update && apt
+                install -y curl
+      terminationMessagePath: /dev/termination-log
+      terminationMessagePolicy: File
+      imagePullPolicy: IfNotPresent
+  restartPolicy: Always
+  terminationGracePeriodSeconds: 30
+  dnsPolicy: ClusterFirst
+  serviceAccountName: default
+  serviceAccount: default
+  securityContext: {}
+  """
+    }
+  }
+
+  stages {
+    stage ('Build Enviroment') {
+      steps {
+        container('compiler') {
+          script {
+            try {
+              echo "---------------- Print environment ----------------"
+              sh "printenv"
+
+              echo "--------------- Building Etendo environment ----------------"
+              sh """
+              git clone $ETENDO_BASE_URL
+              cd etendo_base
+
+              echo -e "context.name=${CONTEXT_NAME}
+              bbdd.sid=${BBDD_SID}
+              bbdd.port=${BBDD_PORT}
+              bbdd.systemUser=${BBDD_SYSTEMUSER}
+              bbdd.systemPassword=${BBDD_SYSTEMPASS}
+              bbdd.user=${BBDD_USER}
+              bbdd.password=${BBDD_PASSWORD}
+              nexusUser=${NEXUS_USER}
+              nexusPassword=${NEXUS_PASSWORD}
+              githubUser=${GITHUB_USER}
+              githubToken=${GITHUB_TOKEN}
+              allow.root=true
+              org.gradle.vfs.watch=false
+              docker_com.etendoerp.docker_db=true
+              org.gradle.jvmargs=-Dfile.encoding=UTF-8" > gradle.properties
+              """
+
+              echo "---------------- Add dependencies ----------------"
+
+              def buildGradleContent = """
+                dependencies {
+                    implementation(\"com.etendoerp.platform:etendo-core:latest.release\")
+                    implementation(\"com.etendoerp:docker:1.1.0\")
```

Comment:
Does this have to pull code instead of dependency? Also, which should not be the fixed version in any case.

Reply:
Fixed! I have changed it to pull the latest commit.